content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 5
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+In standalone mode the Kubelet allows you to manage containers using pod manifests, which brings the benefits of running tightly coupled application as a single unit while leveraging a subset of advanced features including init containers, CNI networking, etc., without the need for the entire Kubernetes feature set.
+
+## {{% heading "objectives" %}}
+
+* Install and configure `containerd`, `cni-plugins`, and `kubelet` on a Linux system.
+* Run `containerd` and `kubelet` as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+
+## {{% heading "prerequisites" %}}
+
+- Admin (root) access to a Linux system (virtual, cloud, or physical instance) running `systemd`.
+- Access to the Internet to download the various components needed for the tutorial.
+- Required CLI tools: `curl`, `tar`.
+- Suggested CLI tools: `jq`.
+
+
+<!-- lessoncontent -->
+
+## Download, install, and configure the components
+
+{{< note >}}
+Download the latest available versions of the required packages.
+The relative path to the downloaded software is assumed to be the `sources` directory.
+Run all the commands as the `root` user.
+{{< /note >}}
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### CNI plugins
+
+Download: [[releases]](https://github.com/containernetworking/plugins/releases)
+
+```shell
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+```
+
+Install:
+
+```shell
+mkdir -p /opt/cni/bin
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+```
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+mkdir -p /etc/cni
+
+cat << EOF > /etc/cni/cni.template
+{
+  "name": "containerd",
+  "cniVersion": "1.0.0",
+  "plugins": [{
+    "type": "ptp",
+    "ipMasq": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "{{.PodCIDR}}",
+      "routes": [
+        {"dst": "0.0.0.0/0"}
+      ]
+    }
+  },{
+    "type": "portmap",
+    "capabilities": {
+      "portMappings": true
+    }
+  }]
+}
+EOF
+```
+
+### containerd
+
+{{% thirdparty-content %}}
+
+Enable IPv4 packet forwarding:
+
+```shell
+cat <<EOF > /etc/sysctl.d/k8s.conf
+net.ipv4.ip_forward = 1
+EOF
+
+sysctl --system
+```
+
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+Download: [[releases]](https://github.com/containerd/containerd/releases)
```

Comment:
```suggestion
Download containerd (a container runtime that implements the Kubernetes 
{{<glossary_tooltip term_id="cri" text="CRI">}}):
```

Reply:
I added the network plugins and container runtime tooltips to the prerequisites section. See if you like it.